### DISCUSSION TITLE
Predefine Master Key for users

### DISCUSSION THREAD

**Discussion by Nilsinger** [keepassxreboot/keepassxc#8182] [Q&A] [answered]
Hello together,

is there a way to configure the Minimum password strength and length for the Master key in the keepassx c.ini?
Furthermore to predefine if upper- or lower cases, numbers and special characters should be used? 

I only found the solution for Keepass 1.x and 2.x

Thanks and best regards
Nils

**Comment by droidmonkey** [comment, marked as answer]
No there is no limit on what is used for database credentials, that is entirely up to the end user. Keepass is a client side application, any settings to this effect are easily defeated and are just safety rails one can jump over if they choose.
